vue-frontend/src/components/contact/NewContactForm.vue

```diff
@@ -0,0 +1,576 @@
+<template>
+  <div class="tw-container tw-my-[50px] lg:tw-mt-0 md:tw-w-full">
+    <form>
+      <div
+        class="tw-rounded-2xl tw-border tw-border-slate-200 tw-bg-white tw-shadow-[0px_0px_45px_rgba(0,0,0,0.1)]"
+      >
+        <div class="tw-py-5 tw-px-8 lg:tw-px-12">
+          <div>
+            <div class="tw-relative tw-pt-6">
+              <input
+                class="tw-block tw-peer tw-my-2 tw-mx-0 tw-w-full tw-box-border tw-border-b tw-border-slate-400 tw-rounded-none tw-px-0 tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.375rem] tw-leading-[1.21rem] md:tw-leading-[2.0625rem] focus:tw-box-border focus:tw-border-b focus:tw-border-slate-400 focus:tw-outline-none placeholder:tw-text-black-core/[0.6]"
+                type="text"
+                name="username"
+                required
+                placeholder=" "
+                autocomplete="given-username"
+                v-model="name"
```

Comment:
There should be space between the label and input like I'll invest field, See the image below

![Screenshot from 2023-02-20 14-25-20](https://user-images.githubusercontent.com/69897605/220058630-0c765a54-10c8-428c-b049-19632dba46cb.png)